x-pack/plugins/security_solution/common/types/detail_panel/index.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { FlowTargetSourceDest } from '../../search_strategy';
+import type { TimelineTabs } from '../timeline';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type EmptyObject = Record<any, never>;
```

Comment:
Would using something like
```
type EmptyObject = {
  [K in string | number]: never;
};
```
reach the same goal? (might save you the es-lint-disable)